wordpress-mobile/WordPress-Android
# Issue/10646 material post settings

Fixes #11253

This PR moves post settings to the material theme and adds a Dark Mode support.

Most of the changes come from XML files, so do not be alarmed by the size.

Location activity is provided by OS as far as I'm aware, and outside of our styling reach.

![comparison_post_settings](https://user-images.githubusercontent.com/728822/73999540-eb5d6d00-4919-11ea-96d8-a96b491f6f25.png)

To test:
- Navigate to post settings and make sure all options and nested screens look ok in both light and dark modes.

You can change the app theme from App Settings.

PR submission checklist:
